{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/components/links/links.ts

```diff
@@ -1,3 +1,5 @@
+import './links.scss'
```

Comment:
we've been treating this as a global handler - see the event listening at the bottom. I'm not sure this is a component becuase it doesn't have an attached jinja file. I think we should either:
a) keep this at the global level
b) remove the event listeners and make this logic per component, and generate a link.jinja file to use as the general link component

Reply:
the link.jinja does exist but looks in need of some TLC/isn't used anywhere/isn't used by the ts or scss files. so probably the scss file could be unlinked and added back to the global scss file

Reply:
moved this to the global level per comments below